src/pages/components/ai-label/usage.mdx

```diff
@@ -0,0 +1,517 @@
+
+<Row>
+<Column colLg={8}>
+
+![A default and inline AI label](images/ai-label-variants.png)
+
+</Column>
+</Row>
+
+<Caption>Default (left), Inline (right)</Caption>
+
+### When to use
+
+#### Mark AI-generated content
+
+AI transparency is key, and the AI label is an accessible, interactive element
+that marks any AI instance presented in the interface.
+
+#### Provides a consistent visual reference point for AI
+
+For recognition and recall, users need a consistent, identifiable visual
+reference to look to when they need to understand more details about how the AI
+was built.
+
+#### A pathway to explainability
+
+Users must clearly understand how to access more details about how an AI was
+built. By providing a bridge to explainability from the AI label, users are
+given a recognizable location to learn more about the AI model.
+
+### When not to use
+
+#### Don’t use the AI label as decoration
+
+These are strictly intended to indicate any time a modality or component is
+using AI.
+
+#### Don’t modify AI label behaviors
+
+The AI label is intended as a pathway to explainability. Any additional
+behaviors can set unclear and inconsistent expectations.
+
+#### Don’t use the AI label as a trigger for AI actions
+
+The AI label is not a UI icon and should not be used with buttons or other
+call-to-actions to trigger an AI action like “regenerate.” Those type of AI
+actions will have their own UI icons.
+
+## Formatting
+
+### Anatomy
+
+<Row>
+<Column colLg={8}>
+
+![Anatomy of AI label](images/ai-label-anatomy.png)
+
+</Column>
+</Row>
+
+1. **Icon**: indicates AI is present and is the button trigger for the
+   explainability popover. It can be either the default or inline variant.
+2. **Text label**: the text abbreviation for artificial intelligence which can
+   be translated as needed.
+3. **Explainability popover**: Area that contains explainability content and
+   interactive elements.
+
+### Sizing
+
+#### Default sizes
+
+The default AI label spans the range of UI icons and pictogram sizes. Which size
+to use is determined by the prominence and hierarchy of the area it is in or by
+its neighboring content like other buttons. When placed next to other ghost icon
+buttons we typically recommend going down one size from the button size so they
+are optically weighted the same.
+
+| Default size     | Height (px/rem) |
+| ---------------- | --------------- |
+| Mini             | 16 / 1          |
+| 2x small (2xs)   | 20 / 1.25       |
+| Extra small (xs) | 24 / 1.5        |
+| Small (sm)       | 32 / 2          |
+| Medium (md)      | 40 / 2.5        |
+| Large (lg)       | 48 / 3          |
+| Extra large (xl) | 64 / 4          |
+
+<Row>
+<Column colLg={8}>
+
+![Default AI label sizes](images/ai-label-sizes-default.png)
+
+</Column>
+</Row>
+
+#### Inline sizes
+
+placed inline with. For example, when inserting the inline AI label into a table
```

Comment:
@aagonzales This statement seems incomplete, is something missing? Was it supposed to read something like this? 

`The AI label is placed inline with the component. `

Reply:
Oh no looks like something got deleted here on accident.